CHANGELOG.md

```diff
@@ -4,6 +4,37 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0-rc3 (2020-11-12)
+
+This release candidate contains bugfixes since the previous release
+candidate, as well as additional minor features including support for
+"user-mapping" authentication between access/data nodes and an
+experimental API for refreshing continuous aggregates on individual
+chunks.
+
+**Minor Features**
+* #2627 Add optional user mappings support
+* #2635 Add API to refresh continuous aggregate on chunk
+
+**Bugfixes**
+* #2593 Set explicitly all lock parameters in alter_job
+* #2604 Fix chunk creation on hypertables with foreign key constraints 
+* #2610 Support analyze of internal compression table
+* #2612 Optimize internal cagg_watermark function
+* #2613 Refresh correct partial during refresh on drop
+* #2617 Fix validation of available extensions on data node
+* #2619 Fix segfault in decompress_chunk for chunks with dropped columns 
+* #2620 Fix DROP CASCADE for continuous aggregate 
```

Comment:
Also need this one I believe: https://github.com/timescale/timescaledb/pull/2560  Fix SCHEMA DROP CASCADE with continuous aggregates